pkg/tfgen/installation_docs.go

```diff
@@ -245,3 +250,106 @@ func convertExample(g *Generator, code string, exampleNumber int) (string, error
 	exampleContent += chooserEnd
 	return exampleContent, nil
 }
+
+type SectionSkipper struct {
```

Comment:
This should not be public.
```suggestion
type sectionSkipper struct {
```